templates/summary.html

```diff
@@ -21,11 +21,10 @@ <h1>{{ _("Check your answers and submit") }}</h1>
 
     <h1>{{content.summary.title}}</h1>
       {% call onsPanel({
-        "type": "success",
```

Comment:
You shouldn't need to define the type as the default is information?

Reply:
Cool, sorted that. I tested it and it works for me